cpp/models/abm/infection.h

```diff
@@ -0,0 +1,142 @@
+/*
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: David Kerkmann, Sascha Korf
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#ifndef EPI_ABM_INFECTION_H
+#define EPI_ABM_INFECTION_H
+
+#include "abm/time.h"
+#include <vector>
+#include "abm/state.h"
+#include <memory>
+
+namespace mio
+{
+namespace abm
+{
+
+class Virus
+{
+public:
+    Virus(double infectivity, double mortality, double recovery)
```

Comment:
Please add issues to allow for different inputs.
- First, arrays for age groups
- Second, different distributions for different age groups (or one distribution for all)

Reply:
This is now handles by the ParameterSet.